docs/document/docs/releases/release-notes.md

```diff
@@ -0,0 +1,8 @@
+---
+id: release-notes
+title: v1.0.4 Release Notes
+sidebar_label: v1.0.4 Release Notes
+original_id: release-notes-v1.0.4
```

Comment:
I think it is no need to specify `original_id` under docs folder.
This is for versioning.

Reply:
Ok, i removed it